app/pages/UploadWrittenAuthorisationPage.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import models.FileAttachment
+
+case object UploadWrittenAuthorisationPage extends QuestionPage[FileAttachment] {
+
+  override def toString: String = "uploadWrittenAuthorisation"
```

Comment:
is this name correct?

Reply:
What do you mean? gram-marly correct, semantically correct or just wrong at all?

Reply:
Ignore this comment.
Sorry, my mistake